src/test/java/com/yelp/nrtsearch/server/luceneserver/DeleteIndexBackupHandlerTest.java

```diff
@@ -84,7 +84,7 @@ public void testDeleteIndexBackupHandlerHandle() throws HandlerException, IOExce
     verify(archiver, never()).deleteVersion(serviceName, resourceName + "_data", versionHash1);
     verify(archiver).deleteVersion(serviceName, resourceName + "_data", versionHash2);
 
-    Assert.assertEquals(response.getDeletedVersionHashesList(), Arrays.asList(versionHash2));
+    Assert.assertEquals(response.getDeletedResourceDataHashesList(), Arrays.asList(versionHash2));
```

Comment:
Would it be possible to assert the other three components in the response too ?

Reply:
Yup, will do